recipes/libcurl/all/conanfile.py

```
@@ -90,11 +90,18 @@ def _is_using_cmake_build(self):
     def _has_zstd_option(self):
         return tools.Version(self.version) >= "7.72.0"
 
+    @property
+    def _has_metalink_option(self):
+        # Support for metalink was removed in version 7.78.0 https://github.com/curl/curl/pull/7176
+        return tools.Version(self.version) < "7.78.0" and not self._is_using_cmake_build
+
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
         if not self._has_zstd_option:
             del self.options.with_zstd
+        if not self._has_metalink_option:
```

Looks like this silently drops the `with_metalink` option.  I think the user should get notified when they have it explicitly set to True